README.md

```diff
@@ -1,24 +1,41 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# PPROJECT: FORMS
 
-Things you may want to cover:
+> Practicing differnt ways to develop forms in Rails
 
-* Ruby version
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby
+- Ruby on Rails
 
-* Database creation
 
-* Database initialization
+## Getting Started
 
-* How to run the test suite
+- Clone the repo.
+- Enter the folder `cd re-former` 
+- Run `bundle install` to make sure that you have all dependencies to run the app.
+- Run `rails db:migrate` 
+- Run `rails c` to open console and interact with application
 
-* Services (job queues, cache servers, search engines, etc.)
+👤 **Author1**
 
-* Deployment instructions
+- GitHub: [@od-c0d3r](https://github.com/od-c0d3r)
+- Twitter: [@od-coder](https://twitter.com/od_coder)
+- LinkedIn: [Profile](https://linkedin.com/in/omarrashad)
 
-* ...
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [required] This link is broken, would you fix it please?

Reply:
[Done] A License file has been created